test/steps/general/3_then.js

```diff
@@ -104,22 +127,22 @@ export function itShouldRejectWithTheOriginalRejection() {
 
 export function itShouldReturnAnEmptyObject() {
 	const { returnValue } = this.test.ctx;
-	return returnValue.should.be.fulfilledWith({});
+	return returnValue.should.be.eventually.eql({});
 }
 
 export function itShouldReturnTrue() {
 	const { returnValue } = this.test.ctx;
-	return returnValue.should.be.true();
+	return returnValue.should.be.true;
 }
 
 export function itShouldReturnFalse() {
 	const { returnValue } = this.test.ctx;
-	return returnValue.should.be.false();
+	return returnValue.should.be.false;
 }
 
 export function itShouldReturnNull() {
 	const { returnValue } = this.test.ctx;
-	return should(returnValue).be.null();
+	return should.equal(returnValue, null);
```

Comment:
This is another downside to switching to chai.

Reply:
But I guess it'll be solved when we switch to `expect`.

Reply:
yes, it will.
I almost tried to port `except` to global, too... lol